### ISSUE TITLE
Add Social Media Profiles to WCA Profiles

### ISSUE THREAD

**Issue by EvilPotato1216** [thewca/worldcubeassociation.org#2441]
We could add social media profiles such as Twitter, Facebook, and YouTube to users' WCA profiles. This could help users interact with each other much easier and know more about the competitors. We could also connect them to the SpeedSolving forum.

**Comment by jfly** [comment]
It's important to note that we are extremely careful about giving people control of the content on the WCA website. Currently, anything our users do that might be seen by other people is always moderated before it goes live (for instance, the WRT must approve every profile picture that is submitted before it goes live).

**Comment by EvilPotato1216** [comment]
Yeah, all that will go on their profile now will be a link to their twitter or facebook page. If anyone wants to contact the competitor or know more about them, all they have to do to go to their social media. Nothing will be shown directly on the website.

**Comment by cubewhiz** [comment]
I have some serious concerns about this--I do not support this at all.

**Comment by EvilPotato1216** [comment]
Why?

**Comment by jfly** [comment]
We don't have any control over these people's social media accounts. They could have very offensive content there that we don't want to link to.

**Comment by Mollerz** [comment]
I agree this is a bad idea. The WCA _isn't_ a social media site, it is a database for competitions and information and shouldn't contain personal social media pages. Similar to ATP Tennis or PSA Squash, it has rankings and player information but no social media links to those individuals.

**Comment by jfly** [comment]
I hear you, @Mollerz. In terms of thinking about our mission as an organization, one very valuable thing the WCA website *can* do for the community is provide people with a way to prove who they are. Think of this as "verified twitter accounts". I agree we need to do a lot of thinking about how we would provide this service to the community, but I think there is value to be added here.

**Comment by EvilPotato1216** [comment]
Is the SpeedSolving Forum a good, clean website. It's not really social media.

**Comment by jbrungar** [comment]
@Mollerz both of those have social media links:

http://www.atpworldtour.com/en/players/rafael-nadal/n409/overview

https://psaworldtour.com/players/view/1099/mohamed-elshorbagy

**Comment by Mollerz** [comment]
Yeah after I posted I had a little dig around and found this out myself, so I did some exploring and thinking about it and I've completely changed my position on it already.

I think it is important to know that these players can be penalised, disallowed entry, or even banned from play based off of things that are posted on their social media. Do we want to expand our regulations to cover things posted on social media for competitors with linked accounts?

Not to mention, these are the best of the best, at what point do you stop allowing people to link their social media accounts? How far down the rankings do you have to go on the atpworldtour site or the psaworldtour site before players stop having social media links? I reckon it isn't that far.

I totally understand the necessity of it actually. Someone like Feliks gets his Twitter, Facebook and YouTube linked to his WCA account. This completely removes the doubt for fake accounts which are very prevalent, especially for huge names like Feliks. Also, sponsors get way more benefit now for sponsoring cubers - the WCA links to a Twitter, Facebook or YouTube account, and almost the first things you see on those pages are the sponsors. It gives them one link from their profile to a lot of external sponsorship opportunities.

I think it's probably best to keep this low priority now, but definitely consider for the future. And when that time comes, only limit it to higher profile cubers that actually have a strong social media presence, and not to let any Tom, Dick or Harry have the capability to add them to their page.

**Comment by EvilPotato1216** [comment]
I guess we could let **everyone** link their SpeedSolving Forum page to their WCA Profile as it's a clean website without any room for harmful content.

**Comment by shelchang** [comment]
I don't like the idea of linking to any external site where we don't have control over the content. Anyone *could* post harmful content to SpeedSolving.

**Comment by Jambrose777** [comment]
I think this can be closed. There are a lot of points about how we limit how much content is allowed on WCA from the individual person (only the Profile Picture) and anything else would generally need to be approved. People can link things in their WCA forum Profile, which should be enough.

**Comment by jonatanklosko** [comment]
I think it's ok to leave this open for discussion, although this is definitely something that requires formal approval if ever implemented.

**Comment by dunkOnIT** [comment]
Closing due to inactivity. FWIW I think there's value in links like YouTube etc, but this is something which needs to be set at a policy level/given to WST as a mandate, not something progress will be made on with further discussion in this issue
